.github/workflows/puzzletron_runtime_image.yml

```
@@ -0,0 +1,128 @@
+          python -m pip install --disable-pip-version-check "packaging>=24,<27"
+          python -m examples.puzzletron.ci.preflight_dependency_metadata \
+            --environment examples/puzzletron/ci_environment.json
+
+  runtime-image-required-check:
+    if: ${{ startsWith(github.ref, 'refs/heads/pull-request/') && always() }}
+    needs: [pr-gate, dependency-metadata-preflight, build-runtime-image]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Report intentionally scoped runtime-image validation
+        if: needs.pr-gate.outputs.run_tests != 'true'
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)